Sources/NostrSDK/RelayRequest.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  RelayRequest.swift
+//  
+//
+//  Created by Joel Klabo on 5/25/23.
+//
+
+import Foundation
+
+struct RelayRequest {
+
+    static let encoder = JSONEncoder()
+
+    static func close(subscriptionId: String) -> String? {
+        let payload = [AnyEncodable("CLOSE"), AnyEncodable(subscriptionId)]
+        return encode(payload: payload)
+    }
+
+    static func event(_ event: NostrEvent) -> String? {
+        let payload = [AnyEncodable("EVENT"), AnyEncodable(event)]
+        return encode(payload: payload)
+    }
+
+    static func count(subscriptionId: String, filter: Filter) -> String? {
+        let payload = [AnyEncodable("COUNT"), AnyEncodable(subscriptionId), AnyEncodable(filter)]
+        return encode(payload: payload)
+    }
+
+    static func req(subscriptionId: String, filter: Filter) -> String? {
```

Comment:
Suggestion:
I'd prefer to use `request` here rather than `req` for easier readability. It's a technical implementation detail that it's actually "REQ" in the request.

Reply:
fixed: 6cd025c